aspnet/fundamentals/servers.rst

```diff
@@ -39,16 +38,16 @@ I've configured the sample project for this article to support each of the diffe
 
 .. literalinclude:: servers/sample/ServersDemo/src/ServersDemo/project.json
 	:lines: 1-17
-	:emphasize-lines: 12-14
+	:emphasize-lines: 14-15
 	:linenos:
 	:language: javascript
 	:caption: project.json (truncated)
 
-The ``run`` command will execute the application via its ``void main()`` method defined in ``program.cs``. In this case, this has been set up to configure and start an instance of ``WebListener``. This is not a typical means of launching a server, but is shown to demonstrate the possibility (the `Music Store sample application <https://github.com/aspnet/MusicStore>`_ also demonstrates this option).
+The ``run`` command will execute the application via its ``void main()`` method defined in ``program.cs``. In this case, this has been set up to configure and start an instance of ``Kestrel``. This is not a typical means of launching a server, but is shown to demonstrate the possibility (the `Music Store sample application <https://github.com/aspnet/MusicStore>`_ also demonstrates this option).
```

Comment:
I think we can remove any discussion of the commands, and launching a web app via program main is the new standard anyways.